PhysicsTools/Heppy/python/analyzers/objects/JetAnalyzer.py

```diff
@@ -218,11 +257,19 @@ def process(self, event):
                 self.jetsIdOnly.append(jet)
 
         jetsEtaCut = [j for j in self.jets if abs(j.eta()) <  self.cfg_ana.jetEta ]
-        self.cleanJetsAll, cleanLeptons = cleanJetsAndLeptons(jetsEtaCut, leptons, self.jetLepDR, self.jetLepArbitration)
-
+        self.cleanJetsAll=[]
+        cleanLeptons=[]
```

Comment:
you don't really need these two lines of code for how the scoping of python works